doc/sphinx/user/extending/benchmarking-run-time.md

```diff
@@ -0,0 +1,31 @@
+# Benchmarking run time
+
+When changing code it can sometimes be useful to if and how much your code changes make the code faster or slower. To test this, one could compile and run aspect-release compiled from the main branch and from the branch with the changed code (e.g. feature branch) and time both. The problems is that is is not very reliable, given that your computer might be doing something on the background while you are running one, but not while you are running the other.
+
+```{note}
+When running timing benchmarks, always run in release mode.
+```
+
+One solution is to this problem is to run both executables after each other in a random order. This is what the program [cbdr](https://crates.io/crates/cbdr) can do for you. It can then also analyze and report back the result.
```

Comment:
```suggestion
One solution to this problem is to run both executables after each other in a random order. This is what the program [cbdr](https://crates.io/crates/cbdr) can do for you. It can also analyze and report back the result.
```